### PR TITLE
Added a bit to the kind quickstart doc

### PR DESCRIPTION
**Pull request by nerdalert**
Added more to validating the stack

### CODE REVIEW

File: docs/deployment/nexodus-service.md [hunk 1]
```diff
@@ -38,12 +38,39 @@ The Makefile will install the https certs. You can view the cert in the Nexodus
...
 Install [`mkcert`](https://github.com/FiloSottile/mkcert) on the agent node, copy the cert from the service running kind (`.certs/rootCA.pem`) to the remote node you will be joining (or viewing the web UI) and run the following.
 
 ```console
 CAROOT=$(pwd)/.certs mkcert -install
 ```
 
+Verify the service by attaching a node using built-in accounts as part of the kubernetes dev overlay build `make run-on-kind` provides.
+
+```console
+# from the nexodus repo directory root ('make dist' for all hardware builds):
```
Comment (russellb) [inline docs/deployment/nexodus-service.md:64]: `make dist` isn't right. That just creates the `dist` directory.

File: docs/deployment/nexodus-service.md [hunk 2]
```diff
@@ -38,12 +38,39 @@ The Makefile will install the https certs. You can view the cert in the Nexodus
...
+make dist/nexctl-linux-amd64
+sudo NEXD_LOGLEVEL=debug dist/nexctl-linux-amd64 --username admin --password floofykittens  https://try.nexodus.127.0.0.1.nip.io
+```
+
+Alternatively, or build the nexctl binary and running a command with it.
+
+```console
+make dist/nexctl-linux-amd64
```
Comment (russellb) [inline docs/deployment/nexodus-service.md:72]: This is the same command you showed in the previous example.

**Review by russellb — CHANGES_REQUESTED** [review]

File: docs/deployment/nexodus-service.md (hunk 1 above)
Comment (nerdalert) [reply docs/deployment/nexodus-service.md]: Thanks!

**Comment by nerdalert** [comment]
Also changed a md-lint complaint that must have been from a recent update: https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md

File: MAINTAINERS.md [hunk 3]
```diff
@@ -4,8 +4,8 @@ The following table lists the names and email addresses of project maintainers.
...
-| Brent Salisbury | bsalisbu@redhat.com |
-| Dave Tucker | dave@dtucker.co.uk |
-| Hiram Chirino | hiram@hiramchirino.com |
-| Russell Bryant | russell.bryant@gmail.com |
\ No newline at end of file
+| Anil Vishnoi | `avishnoi@redhat.com` |
+| Brent Salisbury | `bsalisbu@redhat.com` |
+| Dave Tucker | `dave@dtucker.co.uk` |
+| Hiram Chirino | `hiram@hiramchirino.com` |
+| Russell Bryant | `russell.bryant@gmail.com` |
```
Comment (russellb) [inline MAINTAINERS.md:11]: This is weird. I wasn't able to reproduce it.

File: docs/deployment/nexodus-service.md [hunk 4]
```diff
@@ -38,12 +38,39 @@ The Makefile will install the https certs. You can view the cert in the Nexodus
...
 CAROOT=$(pwd)/.certs mkcert -install
 ```
 
+Verify the service by attaching a node using built-in accounts as part of the kubernetes dev overlay build `make run-on-kind` provides.
+
+```console
+# from the nexodus repo directory root:
+make dist/nexd-linux-amd64
+sudo NEXD_LOGLEVEL=debug dist/nexd-linux-amd64 --username admin --password floofykittens  https://try.nexodus.127.0.0.1.nip.io
+```
```
Comment (russellb) [inline docs/deployment/nexodus-service.md:67]: I meant to put this last time, but another option is running `make run-nexd-container`. It's an easy way to run one or more sandboxed nexd instances.

**Review by russellb — APPROVED** [review]

File: docs/deployment/nexodus-service.md (hunk 4 above)
Comment (nerdalert) [reply docs/deployment/nexodus-service.md:70]: Neat! I didn't know this was there.

File: MAINTAINERS.md (hunk 3 above)
Comment (nerdalert) [reply MAINTAINERS.md:11]: I wasnt able to locally either but assumed it was in today's updates, here is the CI job:
https://github.com/nexodus-io/nexodus/actions/runs/4854970441

**Pull request merged**